css/terminal7.css

```diff
@@ -319,6 +319,9 @@ button.rename-close {
     font-size: 16pt;
     flex: 10 1 auto;
     line-height: 24pt;
+    color: #01fafa;
+    background-color: #3e3446;
```

Comment:
better to use vars for color values. check the top of the file for an example.

Reply:
I've played with the color and I think it's best to go with what we have in the site - #11051E